database/migration/databaseMethods.ts

```diff
@@ -479,6 +481,103 @@ const databaseMethods = {
       })
     }
   },
+  secureDataTables: async () => {
+    const dataTables = (
+      await DBConnect.query({ text: `SELECT table_name FROM data_table;`, rowMode: 'array' })
+    ).rows.flat()
+
+    for (const table of dataTables) {
+      try {
+        await DBConnect.query({
+          text: `ALTER TABLE ${config.dataTablePrefix}${table} ENABLE ROW LEVEL SECURITY;`,
+        })
+      } catch {
+        console.log(`ERROR: Problem securing data table "${table}"`)
+      }
+    }
+  },
+  addDataViewsForLookupTables: async () => {
```

Comment:
Yeah, most of them are one-time. I've just been putting things in here that were getting too long-winded for the main migration script, but it could be organised a bit better. Not quite matching the original intention here -- something to address at some point.